ui/library/src/components/thumbnails/ThumbnailList.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { DocumentContext } from '../../context/DocumentContext';
+import { Thumbnail } from './Thumbnail';
+
+type Props = any;
+
+export const ThumbnailList: React.FunctionComponent<Props> = () => {
+  const { numPages } = React.useContext(DocumentContext);
+
+  return (
+    <div className="reader__thumbnail-list">
+      <ul className="reader__thumbnail-list__list">
```

Comment:
Consider better class names
```suggestion
    <div className="reader__thumbnail-list-wrapper">
      <ul className="reader__thumbnail-list">
```